docs/changelog.md

```diff
@@ -283,25 +320,32 @@ description: |
   - Two small fixes for PDF standard conformance
 
 - Performance
-  - Typst's layout engine is now multi-threaded. Typical speedups are 2-3x for
-    larger documents. The multi-threading operates on page break boundaries, so
+  - Typst's layout engine is now multithreaded. Typical speedups are 2-3x for
+    larger documents. The multithreading operates on page break boundaries, so
     explicit page breaks are necessary for it to kick in.
   - Paragraph justification was optimized with a new two-pass algorithm.
     Speedups are larger for shorter paragraphs and range from 1-6x.
 
 - Command Line Interface
   - Added `--pages` option to select specific page ranges to export
+    <!-- Add accepted format for the argument.
```

Comment:
CLI docs are a separate topic and out of scope here

Reply:
Wait, what? Why? It is still a part of the release and is _a docs_. Do you just mean I should create a separate issue (maybe PR) for this (before the release)?

Reply:
I meant that we have no docs for the CLI as a whole. Writing them now is too much work. Even if someone else did, I would need to do a lot of review/editing. There is already an issue: https://github.com/typst/typst/issues/3765

Reply:
No dedicated docs, but there is, for example, a whole section about it: https://typst.app/docs/reference/introspection/query/#command-line-queries. But I agree that this won't fit for `--pages`. I was actually thinking that adding one sentence showing the accepted format in the changelog would be enough for now.